src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -115,6 +115,8 @@
     <PlatformManifestFileEntry Condition="'$(PgoInstrument)' != ''" Include="clrjit.pgd" IsNative="true" />
     <PlatformManifestFileEntry Include="libclrjit.so" IsNative="true" />
     <PlatformManifestFileEntry Include="libclrjit.dylib" IsNative="true" />
+    <PlatformManifestFileEntry Condition="('$(Configuration)' == 'Debug' or '$(Configuration)' == 'Checked') or ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'arm64')" Include="libclrinterpreter.so" IsNative="true" />
```

Comment:
We often mix-and-match configurations. E.g. build checked coreclr runtime and everything else as release. Does this condition work well with this mix-and-match?

Reply:
Thanks, probably doesn't work with mix-and-match. I replaced it with CoreCLRConfiguration.